pages/linux/compose.md

```diff
@@ -0,0 +1,12 @@
+# compose
+
+> An alias to a run-mailcap's action compose.
+> Originally run-mailcap is used to  mime-type/file.
+
+- Compose action can be used to compose any existing file or new  on default mailcap edit tool:
+
+`compose {{filename}}`
+
+- With run-mailcap:
+
+`run-mailcap --action=compose {{filename}}`
```

Comment:
@rubenvereecken, @ostera 
Do you think about having `run-mailcup` examples here?

IMO this is slightly breaking man-style of our pages.